api/tests/python/end_points/name_requests/test_nr_actions.py

```diff
@@ -782,3 +793,78 @@ def test_draft_patch_resend(client, jwt, app):
 
     # Check NR number is the same because these are PATCH and call change_nr
     assert_field_is_mapped(draft_nr, patched_nr, 'nrNum')
+
+
+draft_input_fields = {
+	'applicants': [
```

Comment:
Please check whether `applicants` is an array or simply an object. I saw this response to a "get NR" ...

![image](https://user-images.githubusercontent.com/10002889/128384125-75b003c8-8080-40b0-b953-8dcac686f6d4.png)

Reply:
Both work when sending the data to create a new temp NR. The json representation sent back will send an object though:
https://github.com/bcgov/namex/blob/main/api/namex/models/request.py#L220

Reply:
OK, thanks for checking!

This conversation can be closed. (I do not have permission to do so.)